### DISCUSSION TITLE
Getting Error when loading 4.27GB ckpt

### DISCUSSION THREAD

**Discussion by Omenizer** [TheLastBen/fast-stable-diffusion#836] [General]
... but also with other bit depths or sizes/versions too. Original 1.5 ckpt loads fine - what am I doing wrong? I successfully trained 3 different models previously...

BTW thank you!!!! for providing the colabs - awesome work 👍

![image](https://user-images.githubusercontent.com/119709270/205410659-6b1d6e35-6f1b-4edd-9553-6fc9fa1773cd.png)

**Comment by TheLastBen** [comment]
Not enough RAM

**Reply by Omenizer** [reply]
On colab pro?

**Reply by TheLastBen** [reply]
If you keep the runtime shape to standard, yes

Set the GPU to standard and the runtime shape to High-RAM

**Reply by Omenizer** [reply]
![image](https://user-images.githubusercontent.com/119709270/205439516-87a7a253-e553-4c18-9642-d79fe53dea8d.png)
It's indeed the RAM. I just learned that buying compute units is not proper Pro, so I can't get more RAM, will need to upgrade.

Do you know why training with 4.27GB checkpoints was possible 2 weeks ago and now it's not anymore? Trained 4 different ones 

Thanks again!

**Reply by TheLastBen** [reply]
I'm working on a fix, will be available soon

**Reply by Omenizer** [reply]
![image](https://user-images.githubusercontent.com/119709270/205459362-6d3ebb17-de86-4f46-bb7c-797ebb6766d1.png)
 Hey, managed to start and complete training with a 7GB ckpt now, but in the end I'm getting this

**Reply by TheLastBen** [reply]
this was fixed in a previous commit
you can manually run `V2=False` and resume the training for  only 10 steps to avoid wasting that trained model

**Reply by Omenizer** [reply]
Hmm I reran and it's still happening, got the colab open - how do I manually run V2=False? Sorry for stupid, and sorry for putting this in discussion instead of issues, I'm new here :)

**Reply by TheLastBen** [reply]
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Reply by Omenizer** [reply]
Ok managed to do it! result 🤗
